README.Rmd

```diff
@@ -87,6 +88,19 @@ Finally, you can calculate perceptual distances to a reference colour, as in
 distance(c("red","green","blue"), "red")
 ```
 
+### Minimal ggplot2 example
+```r
```

Comment:
I think `knitr` requires braces around the `r`. We probably need to also increase the figure height a bit for these cases—otherwise they look a bit squashed

Reply:
Being the gitHub newby I am, I wasn't aware the Rmd gets knitted - is that resulting in the readme's html display people see on your repo? If so, what is the md for, why do we maintain both?